.github/workflows/release-please.yml

```diff
@@ -0,0 +1,81 @@
+on:
+  push:
+    branches:
+      - main
+
+name: release-please
+
+jobs:
+  release-please:
+    # The secret HCLOUD_BOT_TOKEN is only available on the main repo, not in forks.
+    if: github.repository == 'ansible-collections/hetzner.hcloud'
+
+    runs-on: ubuntu-latest
+    outputs:
+      pr-updated: ${{ steps.outputs.outputs.pr-updated }}
+      branch: ${{ steps.outputs.outputs.branch }}
+      version: ${{ steps.outputs.outputs.version }}
+
+    steps:
+      - id: release
+        uses: google-github-actions/release-please-action@v3
+        with:
+          token: ${{ secrets.HCLOUD_BOT_TOKEN }}
+          release-type: simple
+          package-name: hetzner.hcloud
+
+          # We use antsibull-changelog for the actual user-facing changelog.
+          changelog-path: changelogs/dev-changelog.md
+
+          extra-files: |
+            galaxy.yml
+            plugins/module_utils/version.py
+
+      - name: Prepare outputs
+        id: outputs
+        if: steps.release.outputs.pr != ''
+        run: |
+          echo "pr-updated=true" >> "$GITHUB_OUTPUT"
+          echo "branch=${{ fromJSON(steps.release.outputs.pr).headBranchName }}" >> "$GITHUB_OUTPUT"
+          echo "version=$(echo "${{ fromJSON(steps.release.outputs.pr).title }}" | awk '{print $3}')" >> "$GITHUB_OUTPUT"
+
+  antsibull-changelog:
+    runs-on: ubuntu-latest
+    needs: [release-please]
+    if: needs.release-please.outputs.pr-updated
+    steps:
+      - name: Setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - name: Install dependencies
+        run: pip install antsibull-changelog
+
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.release-please.outputs.branch }}
+
+      - name: antsibull-changelog
+        run: antsibull-changelog release --version "${{ needs.release-please.outputs.version }}"
+
+      - name: Check for diff
+        id: antsibull-diff
+        run: |
+          if [[ $(git status --porcelain) ]]; then
+            echo "changed=true" >> "$GITHUB_OUTPUT"
+          fi
```

Comment:
This step seems to be duplicated in the next one, could we merge them together ?

Reply:
Yea for sure, seems like an oversight by me. Though I would prefer to just remove the shell `if` in the next step and use GitHub Actions `if` to skip the steps if not necessary.